skimage/measure/_regionprops.py

```diff
@@ -1269,6 +1275,8 @@ def regionprops(
         Value of greatest intensity in the region.
     **intensity_mean** : float
         Average of intensity values in the region.
+    **intensity_median** : float
+        Value with the median intensity in the region.
```

Comment:
For consistency, I would use "Value of ..." as in the other parameter descriptions. Happy to submit it in a tiny follow-up PR.